test/test_grid.py

```diff
@@ -43,14 +42,31 @@ def test_init_verts(self):
         face_filename = current_path / "meshfiles" / "1face.ug"
         vgrid.write(face_filename)
 
+    def test_init_ds_var_methods(self):
+        """Tests if accessing a variable though self.{NAME} is 
+        equivalent to doing self.ds[{NAME}]
+        """
+        # Variables in UGRID convention
+        path = current_path / "meshfiles" / "outCSne30.ug"
+        grid = ux.open_dataset(path)
+        xr.testing.assert_equal(grid.Mesh2_node_x, grid.ds['Mesh2_node_x'])
```

Comment:
This assertion works correctly for this test case, but FYI: In fact, for every cases, we are accessing the variable through the dictionary as follows:

`grid.ds[grid.ds._var_names["Mesh2_node_x"]]`